packages/block-editor/src/components/use-simulated-media-query/index.js

```diff
@@ -22,9 +23,15 @@ function getStyleSheetsThatMatchHostname() {
 	return filter(
 		get( window, [ 'document', 'styleSheets' ], [] ),
 		( styleSheet ) => {
+			if ( ! styleSheet.href ) {
+				return false;
+			}
+			const matcheableUrl = styleSheet.href.match( URL_REGEX );
 			return (
-				styleSheet.href &&
-				styleSheet.href.includes( window.location.hostname )
+				matcheableUrl &&
+				matcheableUrl.length > 2 &&
+				matcheableUrl[ 1 ] === window.location.protocol &&
```

Comment:
The url package has a `getProtocol` function that might be useful as this could be simplified to `getProtocol( styleSheet.href ) === window.location.protocol`.

Reply:
Oooh great tip, thanks! Updated.